lang/java/reef-tests/src/test/java/org/apache/reef/tests/TestEnvironment.java

```diff
@@ -55,4 +55,6 @@
   int getTestTimeout();
 
   LauncherStatus run(final Configuration driverConfiguration);
+
+  LauncherStatus run(final Configuration runtimeConfiguration, Configuration driverConfiguration);
```

Comment:
In that case, `TestEnvironment` provides no abstraction anymore. You can do what you need to do via:

` return DriverLauncher.getLauncher(runtimeConfiguration).run(driverConfiguration, getTestTimeout());`

Also, you can instantiate a `LocalTestEnvironment` directly, as the test shall only be executed on that runtime (for now).